### PR TITLE
Improve rate-limiting error message in code verification view.

### PR DESCRIPTION
**Pull request by charlesmchen**
Fixes: https://github.com/WhisperSystems/Signal-iOS/issues/1752

Very simple copy fix.  It is very easy to hit the rate limiting in the registration / code verification view.  The old copy used technical jargon the user is unlikely to understand.  The new copy is easier to understand an messages a clearer resolution by recommending a specific amount of time to wait.

PTAL @michaelkirk

### CODE REVIEW

**Comment by charlesmchen** [comment]
See: https://github.com/WhisperSystems/SignalServiceKit/pull/114

**Comment by charlesmchen** [comment]
This PR fixes both "registration" and "code verification" views.

### Before: 

screen shot 2017-02-11 at 4 11 17 pm

### After: 

screen shot 2017-02-11 at 4 23 50 pm

File: Podfile.lock [hunk 1]
```diff
@@ -172,6 +173,6 @@ SPEC CHECKSUMS:
   YapDatabase: b1e43555a34a5298e23a045be96817a5ef0da58f
   ZXingObjC: bf15b3814f7a105b6d99f47da2333c93a063650a
 
-PODFILE CHECKSUM: 48dbf2fb380d626bb799a782dd41b6bf1e466506
+PODFILE CHECKSUM: 367862c422b92ee0f62ed16e3f0480d1510c9f50
 
-COCOAPODS: 1.1.1
```
Comment (michaelkirk) [inline Podfile.lock]: Sorry, but can you update to cocoapods 1.1.1 (or newer, and I'll follow suit). I'm wondering if that's why our xcode proj was re-written in this commit (which I don't really like). Maybe I'm crazy, but I actually try to understand the diffs to the xcode.proj.

**Comment by charlesmchen** [comment]
> can you update to cocoapods 1.1.1

Done.

**Pull request merged**